Testing/Unit/PowerShell/Connection/Connect-ExoHelper.Tests.ps1

```diff
@@ -0,0 +1,25 @@
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath "../../../../PowerShell/ScubaGear/Modules/Connection/ConnectHelpers.psm1") -Function 'Connect-EXOHelper' -Force
+
+InModuleScope ConnectHelpers {
+    Describe -Tag 'Connection' -Name 'Connect-EXOHelper' {
+        BeforeAll {
+            function Connect-ExchangeOnline {}
+            Mock -ModuleName ConnectHelpers Connect-ExchangeOnline -MockWith {}
```

Comment:
I think this needs to be changed in all cases to properly mock call to external dependency.
```suggestion
            Mock Connect-ExchangeOnline -MockWith {}
```